File: Aula03/Go/Vetor02.go
// Faça um programa em golang que receba do usuario
// um vetor com 10 posições. Em seguida devera ser
// impresso o maior e o menor elemento do vetor.

package main

import (
	"fmt"
)

func main() {
	var vetor [10]int

	fmt.Println("Digite 10 números inteiros:")
	for i := 0; i < 10; i++ {
		fmt.Printf("Número %d: ", i+1)
		fmt.Scan(&vetor[i])
	}

	maiorValor := vetor[0]
	menorValor := vetor[0]

	for _, valor := range vetor {
		if valor > maiorValor {
			maiorValor = valor
		}
		if valor < menorValor {
			menorValor = valor
		}
	}

	fmt.Printf("Maior valor: %d\n", maiorValor)
	fmt.Printf("Menor valor: %d\n", menorValor)
}

File: Aula02/Vetor02.py
# Faça um programa em Python que receba do usuario 
# um vetor com 10 posições. Em seguida devera ser 
# impresso o maior e o menor elemento do vetor.

def main():
    vetor = [int(input(f"Digite o elemento {i}: ")) for i in range(10)]
    
    maior = max(vetor)
    menor = min(vetor)

    print(f"O maior elemento do vetor é {maior}.")
    print(f"O menor elemento do vetor é {menor}.")
    
if __name__ == "__main__":
    main()
